src/base/operations/manage_account_role_builder.js

```diff
@@ -0,0 +1,113 @@
+import { default as xdr } from '../generated/xdr_generated'
+import { BaseOperation } from './base_operation'
+import { UnsignedHyper } from 'js-xdr'
+
+export class ManageAccountRoleBuilder {
+  /**
+   * Builds operation for role creation
+   *
+   * @param {object} opts
+   * @param {object} [opts.details] - create role details
+   * @param {array} [opts.ruleIDs] - rules that should be assigned to this account
```

Comment:
```suggestion
   * @param {string[]} [opts.ruleIDs] - rules that should be assigned to this account
```

arrays should be described with types they contain